pkg/runtime/runtime.go

```diff
@@ -133,6 +133,8 @@ type DaprRuntime struct {
 	scopedPublishings      map[string][]string
 	allowedTopics          map[string][]string
 	daprHTTPAPI            http.API
+	grpcExternalAPI        grpc.API
+	grpcInternalAPI        grpc.API
```

Comment:
Please do not define these members for now. I would create new issue to split grpc apis between external and internal later, not now.